manifests/extension/config.pp

```diff
@@ -67,46 +67,61 @@
     $module_path = undef
   }
 
-  $ini_name = downcase($so_name)
+  if $::operatingsystem == 'Ubuntu' and $zend != true and $name == 'mysql' {
+    # Do not manage the .ini file if it's mysql. PHP 7.0 and 7.1 do not have mysql.so
+    # If mysql.ini exists and version is 7.0 or 7.1, then remove it
+    if $php::globals::php_version == '7.0' or $php::globals::php_version == '7.1' {
+      exec { 'Remove_php_mysql_ini':
+        command => "phpdismod mysql; rm -f /etc/php/${php::globals::php_version}/mods-available/mysql.ini",
```

Comment:
I really don't think that an exec() doing rm is the correct approach here. A file resource with ensure => absent is probably the way to go.